README.md

```diff
@@ -1 +1,89 @@
-# React Capstone Project | Waris Haleem
\ No newline at end of file
+# React Group Project: Space Traveler's Hub
+
+> The Moneyster website is built using React, Redux, CoinStats APIs, and it allows you to:
+
+> * See list of 100 crypto currencies.
+> * Check latest information of the currencies.
+
+![Project Screenshot](./snapshot.PNG "MoneySter")
+
+## Built With
+
+- **HTML**
+- **CSS**
+- **JavaScript**
+- **iconscouts**
+- **Gitflow**
+- **React**
+- **Redux**
+- **CoinStats APIs**
+
+[Live Demo](https://moneyster.netlify.app/)
+
+[Project Explaination](https://www.loom.com/share/133e2229537f4b439f19e56c2b9dce52)
+
+## Getting Started
+
+### Steps to run the project
+
+#### 1 - Git Clone.
+
+```
+git clone https://github.com/iAmWaris97/react-capstone.git
+```
+#### 2 - Install node packages.
+
+```
+npm install
+```
+
+#### 3 - run the project
+
+```
+npm start
+```
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+## Author
+### 👨‍💻 Waris Haleem
+- GitHub: [@iAmWaris97](https://github.com/iAmWaris97)
+- Twitter: [@iAmWaris97](https://twitter.com/iAmWaris97)
+- LinkedIn: [Waris Haleem](https://www.linkedin.com/in/waris-haleem/)
+
+## Acknowledgments
+
+> Thanks to all the developers whose code, documentations, and efforts helped us doing this project.
+
```

Comment:
- [ ] In README here at the "Acknowledgements", Please give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).